pennylane/devices/default_clifford.py

```diff
@@ -0,0 +1,726 @@
+            this device.
+        check_clifford (bool): Check if all the gate operations in the circuits to be executed are Clifford. Default is ``True``.
+        max_error (float): The maximum permissible operator norm error for decomposing circuits with non-Clifford gate operations
+            into Cliffordt+T basis. The default is ``None`` as this device currently supports only Clifford simulations.
+        state (str): Describes what should be returned when the device's state is computed with ``qml.state``. Default is
+            "tableau", which makes it return the final evolved Tableau. Alternatively, one may use "state_vector" to obtain
+            the evolved state vector. Note that the latter might not be computationally feasible for larger qubit numbers.
+        seed (Union[str, None, int, array_like[int], SeedSequence, BitGenerator, Generator, jax.random.PRNGKey]): A
+            seed-like parameter matching that of ``seed`` for ``numpy.random.default_rng``, or
+            a request to seed from numpy's global random number generator.
+            The default, ``seed="global"`` pulls a seed from NumPy's global generator. ``seed=None``
+            will pull a seed from the OS entropy.
+            If a ``jax.random.PRNGKey`` is passed as the seed, a JAX-specific sampling function using
+            ``jax.random.choice`` and the ``PRNGKey`` will be used for sampling rather than
+            ``numpy.random.default_rng``.
+        max_workers (int): A ``ProcessPoolExecutor`` executes tapes asynchronously
+            using a pool of at most ``max_workers`` processes. If ``max_workers`` is ``None``,
+            only the current process executes tapes. If you experience any
+            issue, say using JAX, TensorFlow, Torch, try setting ``max_workers`` to ``None``.
+
+    **Example:**
+
+    .. code-block:: python
+
+        num_qscripts = 5
+
+        qscripts = [
+            qml.tape.QuantumScript(
+                [qml.Hadamard(wires=[0]), qml.CNOT(wires=[0, 1])],
+                [qml.expval(qml.PauliZ(0))]
+            )
+        ] * num_qscripts
+
+    >>> dev = DefaultClifford()
+    >>> program, execution_config = dev.preprocess()
+    >>> new_batch, post_processing_fn = program(qscripts)
+    >>> results = dev.execute(new_batch, execution_config=execution_config)
+    >>> post_processing_fn(results)
+    [0.0, 0.0, 0.0, 0.0, 0.0]
+
+    .. details::
+        :title: Clifford Tableau
+        :href: clifford-tableau-theory
+
+        The device represents a state by the inverse of a Tableau
+        (`Aaronson & Gottesman (2004) <https://journals.aps.org/pra/abstract/10.1103/PhysRevA.70.052328>`_)
+        consisiting of binary variable :math:`x_{ij},\ z_{ij}` for all :math:`i\in\left\{1,\ldots,2n\right\}`,
+        :math:`j\in\left\{1,\ldots,n\right\}`, and :math:`r_{i}` for all
+        :math:`i\in\left\{1,\ldots,2n\right\}`.
+
+        .. math::
```

Comment:
Is this, somehow, different than the format [Stim ](https://arxiv.org/pdf/2103.02202.pdf) uses for its Tableau?

Reply:
They use the inverse of this Tableau.